experimental-design/README.adoc

```diff
@@ -48,56 +47,55 @@ image::https://github.com/bigbio/proteomics-metadata-standard/raw/master/experim
+The first column of the SDRF is the **Source Name** (a unique accession of the sample, example - Sample 1). The *Sample* metadata captures the properties of the sample and they are defined as *characteristics*. The sample metadata structure is based on the MAGE-TAB SDRF file format developed by the http://fged.org/projects/mage-tab/[RNA-Seq community].
 
 The Sample metadata has different *Categories/Headings*  to organize all the attributes / column headers of a given Sample:
 
 |===
 |Property        | Mandatory(:white_check_mark:)/Optional(:zero:) | Cardinality | Description | Example
 
 |source  name    | :white_check_mark:             | 1           | The Sample name. | Sample 1
-|characteristics | :white_check_mark: | 0..*      | “Characteristics” column headings should contain an ontology property term in square brackets.*
+|characteristics | :white_check_mark: | 0..*      | “characteristics” column headings should contain an ontology property term in square brackets.*
 
-Multiple _Characteristic_ columns of the same category (e.g., “characteristics[organism part]”) are allowed. Typically the usage implies whole to part from left to right. | Characteristics [Organism Part]
+Multiple _Characteristic_ columns of the same category (e.g., “characteristics[organism part]”) are allowed. Typically the usage implies whole to part from left to right. | characteristics [Organism Part]
 |===
 
-NOTE: * The source database or file for ontology terms in these columns may be given in an adjacent “Term Source” column immediately to the right of the “Characteristics” column. In the absence of a “Term Source” column the value is assumed to be user defined.
-
-Each Sample in the experiment MUST contains a _Source Name_, and a collection of _Characteristics_:
+Each Sample in the experiment MUST contains a _Source Name_, and a collection of _characteristics_:
 
 |===
-| source name | characteristics[organism] | characteristics[organism part] | characteristics[phenotype] | characteristics[phenotype] | factor value[phenotype]
+| source name | characteristics[organism] | characteristics[organism part] | characteristics[compound] | characteristics[phenotype] | factor value[phenotype]
 
-|sample_treat   |homo sapiens |Whole Organism | necrotic tissue      | sample treated with drug A | necrotic tissue
-|sample_control |homo sapiens |Whole Organism | normal               | normal                     | normal
+|sample_treat   |homo sapiens |Whole Organism | necrotic tissue      | compound: drug A | necrotic tissue
+|sample_control |homo sapiens |Whole Organism | normal               | compound: non    | normal
```

Comment:
Under Characteristics[compound], just put "drug A" and "none". Like this: 
| Characteristics[compound] |
|----------------------------|
| drug A |
| none |